Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/Security - IE Site-to-Zone Assignment.json

```diff
@@ -8,7 +8,7 @@
       }
     }
   ],
-  "description": "Add AzureAD URL for Kerberos ticket forwarding to the browser's Intranet zone.",
+  "description": "What is it: Adds Azure AD URL for Kerberos ticket forwarding to the browser's Intranet zone. \n Simeon Notes: Improves your users' browser experience by auto logging into sites on the local intranet. To add or remove sites update the config variable 'AllowSiteToZoneAssignmentList'. \n Learn More: https://support.microsoft.com/en-us/help/182569/internet-explorer-security-zones-registry-entries-for-advanced-users",
```

Comment:
**What does this do?**
Configures URLs to include in the browser's security zones. The baseline configures the Intranet zone to include necessary Microsoft URLs for Azure Active Directory Seamless Single Sign-On. To add or remove additional sites, update the config variable 'AllowSiteToZoneAssignmentList'. 

**Why should you use this?**
Improves your users' browsing experience by automatically logging in to sites secured by Azure AD. 

**What is the end-user impact?**
Users will be unable to configure URLs for browser security zones themselves.

**Learn more**
[Internet Explorer security zones registry entries for advanced users](https://support.microsoft.com/en-us/help/182569/internet-explorer-security-zones-registry-entries-for-advanced-users)